.travis.yml

```diff
@@ -1,46 +0,0 @@
-conditions: v1
-
-sudo: false
-language: python
-
-services:
-- docker
-
-python:
-- 3.6
-- 3.7
-- nightly
-
-jobs:
-  allow_failures:
-  - python: nightly
-
-  include:
-  - stage: Deploy
-    name: Publishing dists to PyPI
-    if: tag IS present
-    python: 3.6
-    script: skip
-    deploy:
-      provider: pypi
-      user: aio-libs-bot
-      password:
-        secure: >
-          hlgl7Ml+yv13wqXFnzHPQKyS0Sj45TmsvC53lRk970TJD8ehfeYSxcdt6UMmxGzGUqGGdakQKUueSFxe4+hCKo1TiabsIPsSJWFXL+RfboZbcGPUVfk0MPNnOEOLBOWnsBVzfRXfWdG6oQ+Sn+2z6WIRcOaiyIFll2ABI1tkVxciDgBRl3czx13V4MFWVEM/8P9YJA7x1srzbVISlWKf1ZOA/nk5xJMyGbsdmEV4JVwFanGCTdALHxi8+8MOwFOxecbHlCSpo2od2JeHZunysHuDt3zX84zwH80LqgUyWoTCYgb5QM213NkZHAokV5JAdPTVQJvsZMk65sELGljN36TcggpeZqDlY6pS5+j116GUHbEqLnZveFYsH2XcqhsY0jcbdzlo9KjlBSBzY0VXXsspZcUVAHIQ+o7Cv/JjfabQTYVAhS4nAZy0P0bWOWfoFZAyFlnN86zePPzx1bqCTyMwlfmeNKgCQov4+OmudgDnFwJtebk4poZHB5U4fPFslNTm7bar6H16nqQ6+/FoXDQ1kiOTSmAZDlcySnPWnVaxxRyeguVTtnPJGNWYChFd/NiGGqYeH9m0GlSDUeW+TUezOLOJgF/UxVupV+ISAcKcgbm28MlQNx8/1IVOhsmY3F199eV3briHI8E3vjO67oBBURove+97s1lK4cS56n0=
-      distributions: sdist bdist_wheel
-      on:
-        tags: true
-        all_branches: true
-        python: 3.6
```

Comment:
@Dreamsorcerer you should also migrate publishing to PyPI. Otherwise, you'll lose the ability to make releases.
I've added a secret called `PYPI_API_TOKEN`, follow pointers at https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ to reimplement this within GHA.